apple-pie/index.html

```diff
@@ -4,36 +4,46 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="/apple-pie/styles/style.css">
```

Comment:
```suggestion
    <link rel="stylesheet" href="styles/style.css">
```
Si le especificas la carpeta, le estas diciendo que desde index.html tiene que haber una carpeta apple-pie, peeero ya estas dentro de ella por eso no hace falta si no estarias buscando apple-pie dentro de apple-pie, luego ya depende de ti poner el ./ o no ponerlo :D